tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "strict": true,
+    "sourceMap": true,
+    "target": "es2018",
+    "module": "esnext",
+    "lib": ["esnext"],
+    "esModuleInterop": true,
+    "moduleResolution": "node",
+    "outDir": "lib",
```

Comment:
I would suggest using dist for compiled version. Also i think rollup should set outdir

Reply:
yes, `outdir` is unnecessary here